src/utils/__tests__/safeRename.spec.js

```diff
@@ -0,0 +1,79 @@
+import childProcess from 'child_process';
+import fsPromises from 'fs/promises';
```

Comment:
We are using [node 14 in CI](https://github.com/rancher-sandbox/rd/blob/5de444952afc10e9bbd82b2144e91a4ac7a975e6/.github/workflows/lint.yaml#L16); as such, this wouldn't exist.  At least, it wouldn't as soon as I fix CI to actually run tests…

Reply:
Offline agreement: v14 supports this.